package.json

```diff
@@ -5,6 +5,7 @@
   "version": "0.11.2",
   "dependencies": {
     "probe-image-size": "^5.0.0",
+    "prompt-sync": "^4.2.0",
```

Comment:
Was this a mistake?

Reply:
You don't really need this, btw. `await Streams.stdin.readLine()` will do the same thing; just wrap your script in an async function.